internal/testutil/acc/attribute_checks.go

```diff
@@ -54,3 +54,23 @@ func JSONEquals(expected string) resource.CheckResourceAttrWithFunc {
 		return nil
 	}
 }
+
+func AddAttrSetChecks(targetName string, checks []resource.TestCheckFunc, attrNames ...string) []resource.TestCheckFunc {
+	// avoids accidentally modifying existing slice
+	newChecks := []resource.TestCheckFunc{}
+	copy(newChecks, checks)
```

Comment:
not sure this copy is doing anything as destination is empty, e.g: https://yourbasic.org/golang/copy-explained/
" It returns the number of elements copied, which will be the minimum of len(dst) and len(src) "

If destination is empty nothing is copied

Reply:
Great. Thank you for the reference as well 🙏

Reply:
Note: I had to add the capacity argument too, otherwise ci-lint complains about `append to slice `newChecks` with non-zero initialized length (makezero)`